app/src/containers/FeedbackContainer/index.js

```diff
@@ -0,0 +1,217 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { graphql } from 'react-apollo';
+import gql from 'graphql-tag';
+import * as FeedbackActionCreators from './actions';
+import cssModules from 'react-css-modules';
+import styles from './index.module.scss';
+import {
+  FeedbackButton,
+  AddFeedbackForm,
+  LoadingIndicator,
+ } from 'components';
+import validation from './validation/index';
+import { reduxForm } from 'redux-form';
+import Footer from 'grommet/components/footer';
+import Layer from 'grommet/components/layer';
+import Box from 'grommet/components/box';
+import Button from 'grommet/components/button';
+import Menu from 'grommet/components/menu';
+import Section from 'grommet/components/Section';
+
+export const addFeedbackFields = [
+  'nameInput',
+  'urlInput',
+  'descriptionInput',
+];
+const ROOT_URL = 'https://udacity-client.herokuapp.com';
```

Comment:
We have a variable in config/index.js called BASE_URL that is dynamic,  Please use that instead of hardcoding the URL in here.

```
import { BASE_URL } from 'config';
```